homeassistant/components/plugwise/__init__.py

```diff
@@ -1,14 +1,12 @@
 """Plugwise platform for Home Assistant Core."""
-import asyncio
-
 import voluptuous as vol
 
 from homeassistant.config_entries import ConfigEntry
 from homeassistant.const import CONF_HOST
 from homeassistant.core import HomeAssistant
 
-from .const import ALL_PLATFORMS, DOMAIN, UNDO_UPDATE_LISTENER
-from .gateway import async_setup_entry_gw
+from .const import DOMAIN
+from .gateway import async_setup_entry_gw, async_unload_entry_gw
 
 CONFIG_SCHEMA = vol.Schema({DOMAIN: vol.Schema({})}, extra=vol.ALLOW_EXTRA)
```

Comment:
Can be removed, not used here.

Reply:
Also, no need to `import voluptuous`, `import logger` and define a logger.

Reply:
✅